spec/lib/vanagon/project/dsl_spec.rb

```diff
@@ -231,6 +232,35 @@
     end
   end
 
+  describe "#custom_build_var" do
+    let(:project_block) {
+"project 'test-fixture' do |proj|
+  proj.custom_build_var \"TEST_VAR='foo'\"
+end"
```

Comment:
You *could* indent them using the `String#undent` method that Vanagon provides, if you wanted to.